addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3272,8 +3273,42 @@ export class OdooEditor extends EventTarget {
                 this._applyCommand('indentList', ev.shiftKey ? 'outdent' : 'indent');
             } else if (closestTag === 'TABLE') {
                 this._onTabulationInTable(ev);
-            } else if (!ev.shiftKey) {
+            } else if (ev.shiftKey) {
+                new Set(getTraversedNodes(this.editable)
+                    .map(node => closestBlock(node).firstElementChild)
+                    .filter(node => node && isEditorTab(node) &&
+                        !getAdjacentPreviousSiblings(node, sibling => (
+                            sibling.nodeType !== Node.TEXT_NODE || !/^[\u200B\s]*$/.test(sibling.textContent)
+                        )).length)
+                ).forEach(tab => {
+                    let {anchorNode, anchorOffset, focusNode, focusOffset} = sel;
+                    const [updateAnchor, updateFocus] = [anchorNode === tab.nextSibling, focusNode === tab.nextSibling];
```

Comment:
This is not a deconstruction and I see no reason to do a double assignment here.